packages/core/package.json

```diff
@@ -61,6 +61,7 @@
     "@libp2p/interface-content-routing": "1.0.2",
     "@libp2p/interface-registrar": "1.1.0",
     "@libp2p/tcp": "1.0.11",
+    "@types/chai": "4.3.3",
```

Comment:
`"semver" / "@types/semver" seem to be missing

Reply:
Fixed in https://github.com/hoprnet/hoprnet/pull/4069/commits/9cb5354da5c5dd5e868e551a235eafe2395378ec